tests/integration/test_system_containers.sh

```diff
@@ -59,6 +59,10 @@ ${ATOMIC} ps | grep -q "test-system"
 ${ATOMIC} ps --json | grep -q "test-system"
 ${ATOMIC} ps --all | grep -q "test-system"
 ${ATOMIC} ps --json --all | grep -q "test-system"
+${ATOMIC} ps --filter id=test-system | grep -q "test-system"
+${ATOMIC} ps --no-trunc | grep -q "test-system"
+${ATOMIC} ps --quiet | grep -q "test-system"
+${ATOMIC} ps -aq --no-trunc --filter=id=test-system | grep -q "test-system"
 
```

Comment:
can we add a test for no matches?

`+${ATOMIC} ps -aq --no-trunc --filter=id=no-test-system | grep -v -q "test-system"`